Sources/SwiftParser/TokenPrecedence.swift

```diff
@@ -192,47 +192,266 @@ enum TokenPrecedence: Comparable {
+
+    case .__owned:
+      self = .exprKeyword
+    case .__shared:
+      self = .exprKeyword
+    case ._alignment:
+      self = .exprKeyword
+    case ._cdecl:
+      self = .exprKeyword
+    case ._Class:
+      self = .exprKeyword
+    case ._compilerInitialized:
+      self = .exprKeyword
+    case ._const:
+      self = .exprKeyword
+    case ._documentation:
+      self = .exprKeyword
+    case ._dynamicReplacement:
+      self = .exprKeyword
+    case ._effects:
+      self = .exprKeyword
+    case ._expose:
+      self = .exprKeyword
+    case ._forward:
+      self = .exprKeyword
+    case ._implements:
+      self = .exprKeyword
+    case ._linear:
+      self = .exprKeyword
+    case ._modify:
+      self = .exprKeyword
+    case ._move:
+      self = .exprKeyword
+    case ._NativeClass:
+      self = .exprKeyword
+    case ._NativeRefCountedObject:
+      self = .exprKeyword
+    case ._noMetadata:
+      self = .exprKeyword
+    case ._nonSendable:
+      self = .exprKeyword
+    case ._objcImplementation:
+      self = .exprKeyword
+    case ._objcRuntimeName:
+      self = .exprKeyword
+    case ._opaqueReturnTypeOf:
+      self = .exprKeyword
+    case ._optimize:
+      self = .exprKeyword
+    case ._originallyDefinedIn:
+      self = .exprKeyword
+    case ._PackageDescription:
+      self = .exprKeyword
+    case ._private:
+      self = .exprKeyword
+    case ._projectedValueProperty:
+      self = .exprKeyword
+    case ._read:
+      self = .exprKeyword
+    case ._RefCountedObject:
+      self = .exprKeyword
+    case ._semantics:
+      self = .exprKeyword
+    case ._spi_available:
+      self = .exprKeyword
+    case ._swift_native_objc_runtime_base:
+      self = .exprKeyword
+    case ._Trivial:
+      self = .exprKeyword
+    case ._TrivialAtMost:
+      self = .exprKeyword
+    case ._typeEraser:
+      self = .exprKeyword
+    case ._unavailableFromAsync:
+      self = .exprKeyword
+    case ._underlyingVersion:
+      self = .exprKeyword
+    case ._UnknownLayout:
+      self = .exprKeyword
+    case ._version:
+      self = .exprKeyword
+    case .accesses:
+      self = .exprKeyword
+    case .addressWithNativeOwner:
+      self = .exprKeyword
+    case .addressWithOwner:
+      self = .exprKeyword
+    case .any:
+      self = .exprKeyword
+    case .assignment:
+      self = .exprKeyword
+    case .associativity:
+      self = .exprKeyword
+    case .attached:
+      self = .exprKeyword
+    case .autoclosure:
+      self = .exprKeyword
+    case .available:
+      self = .exprKeyword
+    case .before:
+      self = .exprKeyword
+    case .block:
+      self = .exprKeyword
+    case .canImport:
+      self = .exprKeyword
+    case .compiler:
+      self = .exprKeyword
+    case .copy:
+      self = .exprKeyword
+    case .convention:
+      self = .exprKeyword
+    case .cType:
+      self = .exprKeyword
+    case .derivative:
+      self = .exprKeyword
+    case .didSet:
+      self = .exprKeyword
```

Comment:
I think all the accessors (`AccessorDeclSyntax`) should have a `declKeyword` precedence since they start an `AccessorDeclSyntax`, which is a declaration. Doesn’t have to be part of this PR though.

Reply:
Added in 3e14cfe3f9575fc8c1e92af5c1436d1635fdedd5